templates/ovncontroller/bin/start-ovsdb-server.sh

```diff
@@ -15,6 +15,8 @@
 # under the License.
 set -ex
 
+source $(dirname $0)/functions
+
```

Comment:
I assume I'm missing something, but how is this functions module used here? I don't see a single thing that refers to any of functions / variables defined there.

Reply:
yes right, my bad. Will cleanup